includes/connection/class-wc-terms.php

```diff
@@ -49,12 +49,20 @@ public static function register_connections() {
 										'toType'        => $tax_object->graphql_single_name,
 										'fromFieldName' => $tax_object->graphql_plural_name,
 										'resolve'       => function( $source, array $args, AppContext $context, ResolveInfo $info ) use ( $tax_object ) {
-											$resolver = new TermObjectConnectionResolver( $source, $args, $context, $info, $tax_object->name );											
-                                            
-											// Get the term ids that are associated with this $source
-											$terms = wp_list_pluck( get_the_terms( $source->ID, $tax_object->name ), 'term_id' );
-											
-											$resolver->set_query_arg( 'term_taxonomy_id', ! empty( $terms ) ? $terms : array( '0' ) );
+											$resolver = new TermObjectConnectionResolver( $source, $args, $context, $info, $tax_object->name );
+
+											// Set up the term_ids array for the query arguments.
+											$term_ids = [];
+
+											// Get the terms that are associated with this $source.
+											$terms = get_the_terms( $source->ID, $tax_object->name );
+
+											// Extract the term ids if there are terms.
+											if ( ! empty( $terms ) ) {
+												$term_ids = wp_list_pluck( $terms, 'term_id' );
+											}
```

Comment:
@jacobarriola There is actually a function for this.
```suggestion

											$term_ids = \wc_get_object_terms( $object_id, $taxonomy, 'term_id' );
```